runtime/rasdump/javadump.cpp

```diff
@@ -1770,12 +1770,55 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 			writeNativeAllocator("Other", i + 1, 0, frame->liveBytes, frame->liveAllocations);
 		}
 	}
+        /* Write the section trailer */
+        _OutputStream.writeCharacters(
+                "NULL\n"
+                "NULL           ------------------------------------------------------------------------\n"
+        );
+
+#ifdef J9ZTPF
+	if (mmap(NULL, 0, 0, MAP_SUPPORTED, 0, 0) != MAP_FAILED)  {
+		rc = mprivatestats(&ztpfNativeStats);
```

Comment:
I don't see declarations for `rc`, `mprivatestats()` or `ztpfNativeStats`.

Reply:
Apologies, I had a backlevel build which I was doing testing with so I copied only the summary block to my build and verified compilation and contents with that build.  I forget to copy the includes and declarations blocks which were already in my build version into the submission clone (which didn't have that code).

Re-tested after making the review updates and looks good.